serialization.md

```diff
@@ -135,6 +141,20 @@ Address/Hash Helpers
                                                    , 192
                                                    )
                                  )
+
+    rule [hashFakeTxT1]: #hashUnsignedTxT1(TYPE, CID, TN, TP, TG, TT, TV, TD, [TA])
+                      => Keccak256(       #rlpEncodeBytes(TYPE,1)
+                                  +String #rlpEncodeLength(         #rlpEncodeWord(CID)
+                                                            +String #rlpEncodeWord(TN)
+                                                            +String #rlpEncodeWord(TP)
+                                                            +String #rlpEncodeWord(TG)
+                                                            +String #rlpEncodeAccount(TT)
+                                                            +String #rlpEncodeWord(TV)
+                                                            +String #rlpEncodeString(#unparseByteStack(TD))
+                                                            +String #rlpEncodeAccessList([TA], "")
+                                                          , 192
+                                                          )
+                                  )
 ```
```

Comment:
It seems like here, again, we should just pass in `TYPE` as a parameter to the original `#hashUnsignedTx`, and then have it branch on whether `TYPE ==Int 0` or `TYPE ==Int 1`, instead of making another function symbol.